src/navigators/TabNavigator.tsx

```diff
@@ -78,37 +40,51 @@ export function TabNavigator() {
       inactiveColor={theme.colors.onSurfaceDisabled}
       labeled
     >
-      {navigationTabs
-        ? navigationTabs?.map((tab) => (
-            <Tab.Screen
-              name={tab.name}
-              component={tab.component}
-              key={tab.name}
-              options={{
-                tabBarLabel: tab.options.tabBarLabel,
-                tabBarIcon: tab.options.tabBarIcon,
-                headerShown: false,
-              }}
-            />
-          ))
-        : defaultTabs()}
-      {!useTabBar &&
```

Comment:
By the looks of it, the `useTabBar` conditional was to prevent an out-of-bounds error for additional tabs but I've updated the TabBar to force additional tabs to use the same styles from the last tab.